systemtest/src/main/java/io/strimzi/systemtest/performance/PerformanceConstants.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.performance;
+
+/**
+ * Interface for keep global constants used across performance tests.
+ */
+public interface PerformanceConstants {
+
+    /**
+     * Topic Operator specific metrics gathered after test execution
+     */
+    String NUMBER_OF_TOPICS = "Number of Topics";
+    String NUMBER_OF_CLIENT_INSTANCES = "Number of Client Instances";
+    String NUMBER_OF_MESSAGES = "Number of Messages";
+    String CREATION_TIME = "Creation Time";
+    String SEND_AND_RECV_TIME = "Send And Recv Time";
+    String DELETION_TIME = "Deletion Time";
+    String TOTAL_TEST_TIME = "Total Test Time";
+    String MAX_BATCH_SIZE = "Max Batch Size";
+    String MAX_BATCH_LINGER_MS = "Max Batch Linger Ms";
+    String BOB_UPDATE_TIMES = "Bob Update Times";
+    String BOB_NUMBER_OF_TOPICS_TO_UPDATE = "Bob Update Number of Topics";
```

Comment:
yeah i understand it is for use case there are also some more references in names like 
```
String TOPIC_OPERATOR_BOBS_STREAMING_USE_CASE = "bobStreamingUseCase";
String TOPIC_OPERATOR_ALICE_BULK_USE_CASE = "aliceBulkUseCase";
```
which makes it clear so i guess this can stay.